libraries/botbuilder-dialogs-adaptive-testing/tests/resources/LGGeneratorTests/ManuallySetLocale.test.dialog

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
This file is different from the corresponding [.NET `manuallySetLocale.dialog` file](https://github.com/microsoft/botbuilder-dotnet/blob/00d1155d89186d3d6378d5df2fee099e00ed0a4f/tests/Microsoft.Bot.Builder.Dialogs.Adaptive.Templates.Tests/Fallback/manuallySetLocale.dialog), is this okay?